requirements.txt

```diff
@@ -8,3 +8,5 @@ h5py
 matplotlib
 packaging
 pyyaml
+plotly
```

Comment:
Can we make these conditional imports inside the completeness module, SKDH already imports matplotlib and I'd prefer to keep requirements for another plotting module contrained if they are only being used for one module. Same with `avro` below

Reply:
We can, but do we create a new requirements.txt file inside the module or how do we do that?

Reply:
No, but you do something like:
```python
def predict(...):
    try:
        import plotly
    except ImportError:
        raise ImportError("plotly is required for this module")
```

Reply:
Removed avro and plotly from requirements.txt and added a try-except statement for each of those instead